### ISSUE TITLE
Support For User Mapping

### ISSUE THREAD

**Issue by ashu-savani** [geoffdutton/amplitude#27]
After reviewing the README and the NodeJS module page, I couldn't find any documentation indicating that the API supports user mapping/aliasing (https://help.amplitude.com/hc/en-us/articles/360002750712#h_76557c8b-54cd-4e28-8c82-2f6778f65cd4). I just wanted to confirm that it doesn't support this?

**Comment by geoffdutton** [comment]
No, it currently doesn't support it. Are you interested in making a PR?
